openupgrade_scripts/scripts/google_account/16.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,5 @@
+---Models in module 'google_account'---
+---Fields in module 'google_account'---
+---XML records in module 'google_account'---
+DEL ir.config_parameter: google_account.config_google_redirect_uri (noupdate)
```

Comment:
shouldn't we use delete_records_safely_by_xml_id()?

Reply:
Oop miss this, i'll update , thanks

Reply:
Done sir